wallet/src/de/schildbach/wallet/ui/InputParser.java

```diff
@@ -423,27 +440,6 @@ protected void cannotClassify(final String input) {
         error(null, R.string.input_parser_cannot_classify, input);
     }
 
-    public static void dialog(final Context context,
```

Comment:
`dialog()` methods weren't used and in any case, `InputParser` shouldn't be responsible for dialogs.